src/app/workspace/resource/resource-instance-form/select-resource-class/select-resource-class.component.html

```diff
@@ -12,5 +12,8 @@
     <mat-form-field class="large-field resLabel">
         <mat-label>Resource label</mat-label>
         <input matInput formControlName="label" [value]="label" class="label">
+        <mat-error *ngIf="form.controls.label.hasError('pattern')">
+            <span class="custom-error-message">Your label must contain and start with at least one character or one number. Special characters allowed.</span>
```

Comment:
"Your label must start with a letter or a number and cannot contain spaces."

I think it should be obvious to a user that they can use special characters except for the first character (which we warn them about). They will try to use them anyways if they want them 😅 Is the "no spaces allowed anywhere in the label" intended or does the regex need to be modified?

Reply:
fixed in df6a7ae